academy/4-my-own-chain/game-wager.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: The Game Wager Elements
+title: Playing with Money
 order: 15
 description: You let players play with money
 tag: deep-dive
 ---
 
-# The Game Wager Elements
+# Playing with Money
```

Comment:
Working with Money?

Reply:
Could do. The problem may be more with money though.